R/create_ensemble_relative_skill.R

```diff
@@ -55,10 +55,9 @@ create_ensemble_relative_skill <- function(forecasts,
   # include only models with forecasts,
   #   with evaluation for >= x weeks
   skill <- evaluation %>%
-    select(model, continuous_weeks, target_variable,
```

Comment:
We're not filtering on continuous weeks anymore? Or this step has been moved elsewhere?

In any case, the comments in lines 55-56 should be deleted as well.